runconfig/inspect.go

```diff
@@ -0,0 +1,48 @@
+package runconfig
+
+import (
+	"time"
+)
+
```

Comment:
not sure about creaating this auxiliary type just to use it in container inspect daemon response (my opinion of course)
Basically here we are duplicating what's in container struct, to me this should go to api/types at least